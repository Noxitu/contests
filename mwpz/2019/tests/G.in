2
1 5 10
17 7 27