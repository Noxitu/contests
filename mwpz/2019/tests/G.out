16 31 21 19 20
253 358 30 44 15 118 30